doc/modules/mixture.rst

```diff
@@ -195,6 +168,44 @@ to some mixture components getting almost all the points while most
 mixture components will be centered on just a few of the remaining
 points.
 
+.. figure:: ../auto_examples/mixture/images/plot_bayesian_gaussian_mixture_001.png
+   :target: ../auto_examples/mixture/plot_bayesian_gaussian_mixture.html
+   :align: center
+   :scale: 50%
+
+.. topic:: Examples:
+
+    * See :ref:`plot_bayesian_gaussian_mixture.py` for a comparaison of
+      the results of the ``BayesianGaussianMixture`` for different values
+      of the parameter ``alpha``.
+
+Pros and cons of class :class:`BayesianGaussianMixture`: variational inference
```

Comment:
Please remove the first "class" in this header. Actually it could be better rephrased to:

Pros and cons of variational inference with :class:`BayesianGaussianMixture`